### ISSUE TITLE
Does not work on Visual Studio Code

### ISSUE THREAD

**Issue by Frost1z** [archagon/sensible-side-buttons#81]
Thanks for such a convenient app.

I notice that it does not work on Visual Studio Code, hoping for a fix.

Using:
SensibleSideButtons 1.0.6
Visual Studio Code 1.85.0 (Universal)
MacOs Ventura 13.5

**Comment by Max-Laser** [comment]
Yes Actually without the application sensible-side-buttoms running everything is fine. But if you start it it blocks the usage of getting forward and back

**Comment by alexanderczigler** [comment]
The reason it does not work is that Sensible Side Buttons simulates three-finger swipe, whereas Visual Studio Code is looking for mouse buttons 4 and 5 to navigate back and forward.

A potential solution could be to use something like [Swipe To Navigate VSCode Extension](https://marketplace.visualstudio.com/items?itemName=Seivan.swipe-to-navigate).

**Comment by rga-coding** [comment]
Any updates on this? 
https://marketplace.visualstudio.com/items?itemName=Seivan.swipe-to-navigate didn't work for me

**Comment by nan-af** [comment]
This has been fixed in a recent VS code release: https://github.com/microsoft/vscode/issues/204812#issuecomment-3549148303

But it's not the same thing, as it switches between tabs instead of going back/forward in history. So if you need that I guess your bet bet would be to use [this](https://github.com/thealpa/SaneSideButtons) fork which has support for turning it off on a per-app basis
